vms/job/services.py

```diff
@@ -7,7 +7,6 @@
                             get_volunteer_shifts_with_hours,
                             get_unlogged_shifts_by_volunteer_id)
 
-
```

Comment:
Why this deletion? :thinking:

Reply:
corrected Thanks

Reply:
this is still shown, have you made the change?